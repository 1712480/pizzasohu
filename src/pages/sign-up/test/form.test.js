import React from 'react';
import renderer from 'react-test-renderer';
import Form from '../form';

test('Snap shot', () => {
    let snap = renderer.create(
        <Form />
    ).toJSON();

    expect(snap).toMatchSnapshot();
})