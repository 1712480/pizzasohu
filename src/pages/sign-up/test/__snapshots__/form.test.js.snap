// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snap shot 1`] = `
<div
  className="form-container"
>
  <form
    action="#"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="title"
    >
      Member sign up
    </div>
    <div
      className="section"
    >
      <div
        className="title"
      >
        Personal Information
      </div>
      <label>
        First name:
      </label>
      <input
        name="firstName"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
      <label>
        Last name:
      </label>
      <input
        name="lastName"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
      <label>
        Birthday:
      </label>
      <input
        name="birthDay"
        onBlur={[Function]}
        onChange={[Function]}
        type="date"
        value=""
      />
    </div>
    <div
      className="section"
    >
      <div
        className="title"
      >
        Contact Information
      </div>
      <label>
        Email:
      </label>
      <input
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
      <label>
        Phone number: 
      </label>
      <input
        name="phoneNumber"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
      <label>
        Address:
      </label>
      <input
        name="address"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="section"
    >
      <div
        className="title"
      >
        Passwords
      </div>
      <label>
        Password:
      </label>
      <input
        name="password"
        onBlur={[Function]}
        onChange={[Function]}
        type="password"
        value=""
      />
      <label>
        Re-enter password:
      </label>
      <input
        name="passwordConfirmation"
        onBlur={[Function]}
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <div
      className="agreement"
    >
      By clicking Sign up, you agree to our Terms and Data Policy, including our Cookie Use.
    </div>
    <button
      type="submit"
    >
      Sign up
    </button>
  </form>
</div>
`;
